tests/helpers/unittest_base.py

```diff
@@ -220,6 +223,11 @@ def unique_participant_biobank_id(self):
         self._next_unique_participant_biobank_id += 1
         return next_biobank_id
 
+    def unique_questionnaire_response_id(self):
```

Comment:
There is the `QuestionnaireTestMixin` class that encapsulates Questionnaire functions for unittests, why did you not put all these Questionnaire functions in that class?